src/V3Case.cpp

```diff
@@ -227,15 +228,51 @@ class CaseVisitor final : public VNVisitor {
                 }
             }
         }
-        for (uint32_t i = 0; i < numCases; ++i) {
-            if (!m_valueItem[i]) {
-                nodep->v3warn(CASEINCOMPLETE, "Case values incompletely covered "
-                                              "(example pattern 0x"
-                                                  << std::hex << i << ")");
-                m_caseNoOverlapsAllCovered = false;
-                return false;
+        if (AstEnumDType* enumDtp = VN_CAST(nodep->exprp()->dtypep()->skipRefToEnump(), EnumDType)) { // CASE is enum
```

Comment:
As this function is getting large, I suggest move all of this to a "bool caseIsEnumComplete(AstCase* nodep, ...)" subroutine.

Reply:
This was fixed in 299ad459343eb7f1b6c2ac367a41c1414df366df